sdc/tests/tests_perf/generator.py

```diff
@@ -0,0 +1,95 @@
+import time
+from sdc.io.csv_ext import to_varname
+
+
+def gen(cases, method, class_add, type, prefix=''):
+    for params in cases:
+        if len(params) == 4:
+            func, param, length, call_expression = params
+        elif len(params) == 3:
+            func, param, length = params
+            call_expression = None
+        else:
+            func, param = params
+            call_expression = None
+            length = None
+        name = func
+        if param:
+            name += "_" + to_varname(param)
+        func_name = 'test_{}_{}_{}'.format(type, prefix, name)
```

Comment:
Prefix is with `.`? Could you remove it?

Reply:
Prefix is '' or 'pref.'.

Reply:
This name line need, because this writing in file with results

Reply:
I mean prefix usually is `str.`. So test name will be `test_series_str._min`. Is it correct? Please, check.

Reply:
I see. `to_varname("test_series_str._min")` -> `"test_series_str__min".replace("__", "_")` -> `"test_series_str_min"`.
Ok. But it is not obvious.